### DISCUSSION TITLE
Temp 1 and 2 Drive size with 2.0

### DISCUSSION THREAD

**Discussion by PuNkYsHuNgRy** [Chia-Network/bladebit#210] [Q&A]
Hello, I was wondering what the temp sizes needed are for Temp Drive #1 and #2, I am having trouble allocating buffers with 2.0 but can create ramdisks easily. I wanted to try to create ramdisks for Temp #1 and #2 and try it that way.

**Comment by harold-b** [comment]
I believe temp 1 is currently around `300-318GiB`. Temp 2 is around `192GiB` or around `100GiB` if you use the `-a` switch.
There values might shift a little bit on upcoming releases.

**Temp 1** only does sequential writes that will not be used until the next phase, so using a ramdisk will likely not give you much gains as opposed to an SSD.

If you are using a ramdisk you need to disable direct IO on that target dir. You can do that by using `--no-t1-direct` and `--no-t2-direct` for temp1 and temp2 respectively.

You can also just use the in-app cache for temp2 instead of ramdisk.
